src/Components/Users/UserModifierForm.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import '../FormStyles.css';
+import { useFormik } from 'formik';
+import axios from 'axios';
+
+const UserModifierForm = ({user}) => {
+
+    const hasUser = !!user;
+
+    const handleOnSubmit =  async (values) => {
+        try{
```

Comment:
One benefit of then().catch() is that you can chain functions and make the code more redeable and clean than using try{} catch{}. Additionally in this case you are catching errors but you are not doing anything with that error, that is not a good practice. Additionally, other benefit of then().catch() its the bug propagation, if you chain multiple functions the catch() will catch() any of the previous functions errors in a simple and clear way. 
Remember that the code should be the clearest as you can.

Reply:
and finally the try/catch should be done in the service because its business logic

Reply:
Already change it. Anyway you can chain try/catch, if you chain .then().catch() its more lines, you can also use multiples catch to prevent bug propagation.